src/pages/History/HistoryTable/index.tsx

```diff
@@ -0,0 +1,166 @@
+    newColumns.splice(3, 0, liquidityCol);
+    return newColumns;
+  }, [columns]);
+
+  // Format assets to rows
+  const rows: TableProps['data'] = transactions.map(txn => [
+    {
+      key: 'id',
+      render: () => <Typography variant="small2">{txn.id}</Typography>,
+      value: txn.id,
+      align: 'left',
+    },
+    {
+      key: 'type',
+      render: () => (
+        <Token
+          variant="small2"
+          css={styles.whiteText}
+          symbol={getTokenIdFromVAddress(txn.vTokenAddress) as TokenId}
+        />
+      ),
+      value: txn.vTokenAddress,
+      align: 'left',
+    },
+    {
+      key: 'txnHash',
+      render: () => (
+        <EllipseText css={styles.txnHash} text={txn.transactionHash}>
+          <Typography
+            className="ellipse-text"
+            component="a"
+            href={generateBscScanUrl(txn.transactionHash, 'tx')}
+            target="_blank"
+            rel="noreferrer"
+            variant="small2"
+            css={styles.txnHashText}
+          />
+        </EllipseText>
+      ),
+      value: txn.transactionHash,
+      align: 'left',
+    },
+    {
+      key: 'block',
+      render: () => <Typography color="textPrimary">{txn.blockNumber}</Typography>,
+      value: txn.blockNumber,
+      align: 'left',
+    },
+    {
+      key: 'from',
+      render: () => (
+        <EllipseText css={styles.txnHash} text={txn.from}>
+          <Typography
+            className="ellipse-text"
+            component="a"
+            href={generateBscScanUrl(txn.from, 'address')}
+            target="_blank"
+            rel="noreferrer"
+            variant="small1"
+            css={styles.txnHashText}
+          />
+        </EllipseText>
+      ),
+      value: txn.from,
+      align: 'left',
+    },
+    {
+      key: 'to',
+      render: () => (
+        <EllipseText css={styles.txnHash} text={txn.to}>
+          <Typography
+            className="ellipse-text"
+            component="a"
+            href={generateBscScanUrl(txn.to, 'address')}
+            target="_blank"
+            rel="noreferrer"
+            variant="small1"
+            css={styles.txnHashText}
+          />
+        </EllipseText>
+      ),
+      value: txn.to,
+      align: 'left',
+    },
+    {
+      key: 'amount',
+      render: () => (
+        <Typography variant="small1" css={styles.whiteText}>
+          {formatCoinsToReadableValue({
+            value: txn.amount,
+            tokenId: getTokenIdFromVAddress(txn.vTokenAddress) as TokenId,
+            shorthand: true,
+            symbol: false,
+          })}
+        </Typography>
+      ),
+      value: txn.amount.toFixed(),
+      align: 'right',
+    },
+    {
+      key: 'createdAt',
+      render: () => (
+        <Typography variant="small1" css={styles.whiteText}>
+          {t('history.createdAt', { date: txn.createdAt })}
+        </Typography>
+      ),
+      value: txn.createdAt.toDateString(),
```

Comment:
Since this column needs to be orderable:
```suggestion
      value: txn.createdAt.getTime(),
```